src/core/textrenderer/qgstextbackgroundsettings.cpp

```diff
@@ -22,6 +22,7 @@
 #include "qgspainting.h"
 #include "qgstextrendererutils.h"
 #include "qgspainteffectregistry.h"
+#include "qgsapplication.h"
```

Comment:
Out of curiosity, why adding this to a couple of files here?

Reply:
some much heavy includes where pulled out of header files this links to, which incidentally included qgsapplication.